src/api/queues.js

```diff
@@ -213,6 +213,17 @@ router.delete(
   '/:queueId',
   [requireCourseStaffForQueue, requireQueue, failIfErrors],
   safeAsync(async (req, res, _next) => {
+    const { id: queueId } = res.locals.queue
+    const questions = await Question.findAll({
+      where: {
+        queueId
+      }
+    })
+    questions.map(async function (question) {
```

Comment:
This would probably be better done as a bulk update: check out http://docs.sequelizejs.com/manual/tutorial/querying.html and search for "bulk update". In general, 1 query is more efficient than n queries 🙂